src/nidcpower/unit_tests/test_nidcpower.py

```diff
@@ -0,0 +1,319 @@
+import cmath
+import pytest
+
+import nidcpower
+
+
```

Comment:
Add unit tests to validate the math in the properties of `LCRMeasurement`.

Reply:
I suggested elsewhere to remove the math and that way we don't need to validate it.
All the math is done by the driver runtime and passed in in `data`.

I suggest just keeping `data` as a private member and implementing the exposed properties in term of that. That way members aren't creating until and only if needed.

Reply:
I have followed Marcos' suggestion to store `data` as a private member and removed all the calculations in this class.